task_8_9_10.py

```diff
@@ -0,0 +1,24 @@
+from utils import print_intro
+
+print_intro()
+
+
+def get_position():
+    while True:
+        position = input('Enter the position of the chess piece (e.g. a1): ')
+        if len(position) != 2 or position[0] not in 'abcdefgh' or position[1] not in '12345678':
+            print('Invalid input. Please enter a letter between a-h and a number between 1-8.')
+        else:
+            return position[0], int(position[1])
+
+
+# pylint: disable=invalid-name
+def determine_color(let, num):
+    color_cell = 'black' if (ord(let) + num) % 2 == 0 else 'white'
```

Comment:
so, the issue was that you applied **ord**  on letter here, but **int** on number in the previous function.
try to keep similar functionality together.